### PR TITLE
create class for storing security indicators

### PR DESCRIPTION
**Pull request by KPostOffice**
Store security indicator results

### CODE REVIEW

File: thoth/storages/graph/security_indicators.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Adapter for storing security indicators (SI) onto a persistence remote store."""
+
+from .result_base import ResultStorageBase
+
+
+class SecurityIndicatorResultsStore(ResultStorageBase):
+    """Adapter for persisting recommendation results."""
```
Comment (pacospace) [inline thoth/storages/graph/security_indicators.py:24]: `"""Adapter for persisting security indicators results."""

**Comment by pacospace** [comment]
> Do we want to have specific adapters per security indicator result type? (bandit, cloc, ..)

I thought everything could go to one security-indicator, but they produce different outputs probably difficult to be generalized. So maybe it's a good idea to have separate adapters.

File: thoth/storages/si_bandit.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Adapter for storing bandit security indicators (SI) onto a persistence remote store."""
+
+from .result_base import ResultStorageBase
+
+
+class SecurityIndicatorResultsStore(ResultStorageBase):
```
Comment (pacospace) [inline thoth/storages/si_bandit.py:23]: class names should be changed as well :)

Comment (fridex) [reply thoth/storages/si_bandit.py:23]: And please [add them also to the `__init__.py` file](https://github.com/thoth-station/storages/blob/master/thoth/storages/__init__.py) so we can easily import them.

**Comment by fridex** [comment]
recheck

**Pull request merged**